libcfnet/server_code.c

```diff
@@ -65,11 +68,16 @@ static int OpenReceiverChannel(void)
         .ai_socktype = SOCK_STREAM
     };
 
+    if (bind_address == NULL)
+    {
+        bind_address = BINDINTERFACE;
+    }
+
     /* Listen to INADDR(6)_ANY if BINDINTERFACE unset. */
     char *ptr = NULL;
-    if (BINDINTERFACE[0] != '\0')
+    if (bind_address[0] != '\0')
     {
-        ptr = BINDINTERFACE;
+        ptr = bind_address;
         query.ai_flags |= AI_NUMERICHOST;
     }
 
```

Comment:
STUPID github won't let me comment on code below this line. There's one more instance of you should update, right?

```

Reply:
Yes, good catch! Fixing.